DirectEmployers/MyJobs
# [PD-3884] Include deleted comm recs in comm rec reports

## What's this PR do?
Communication records that have been archived do not currently display in the communication record report.

Display these records, along with an archived date so that the user can tell the record is archived.

## Screenshots (if appropriate)

## Where should the reviewer start?
PRM or reporting.

## How should this be manually tested?
If you already have archived comm recs, head to reporting and run a report.
If not, delete a contact with comm recs, or a few comm recs individually.
Ensure the reports now contain the deleted comm recs.

## Any background context you want to provide?
There is a small change to schema.py that removes old partner assessment code.

## PR Checklist

### Separation of Concerns


### Tests


### Migrations
